pkg/client/client.go

```diff
@@ -507,15 +510,16 @@ func (c *Client) Fetch(ctx context.Context, request FetchRequest) (res *FetchRes
 							pLog.Warn("received partial fetch error", parsePartialFetchKV(fetchError)...)
 						}
 						fetchSummaries <- ProviderFetchSummary{
-							ProviderName:          providerConfig.Name,
+							ProviderName:          providerConfig.Alias,
```

Comment:
I think this is for diagnostics and not related to SaveFetchSummary?

Reply:
current configuration allows to create more than one instance of a provider only in case if instances have different aliases. Also fetch summaries look wrong if we use only provider name since only results of latest fetch summary are displayed in console.

Reply:
![image](https://user-images.githubusercontent.com/20381690/150340462-8e1fc23e-8a18-4065-bfa0-4f7131acb19d.png)
here how output looks now when we use 2 aliases
before it was only 1 line which is wrong